src/main/java/frc/robot/subsystems/conveyor/commands/Convey.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot.subsystems.conveyor.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.conveyor.Conveyor;
+
+/**
+ * This command convey the balls to the shooter, or to the funnel.
+ */
+public class Convey extends CommandBase {
+    private final Conveyor conveyor;
+    private final double power;
+    private final boolean toShooter;
+
+    public Convey(Conveyor conveyor, double power, boolean toShooter) {
+        this.conveyor = conveyor;
+        this.power = power;
+        this.toShooter = toShooter;
```

Comment:
Why not just a negative `power` if you want to bring Power Cells back inside?

Reply:
I don't want to bring a power cell back to the conveyor, I might want to get the power cell out of the robot. I don't want to use this mechanism because I might have a different `power` to apply when we use it in conjunction with the funnel

Reply:
You didn't answer my questions...

Reply:
You didn't answer mine...
Either way, I'm going to split the command into different commands, because this command handles a lot of situation that aren't related to each other

Reply:
> You didn't answer mine...

Did I miss anything? All question marks in this conversation are mine 😁.

Reply:
In this conversation yes, but not in this PR
I asked [this](https://github.com/Galaxia5987/robot-2021/pull/9#pullrequestreview-566206941) 17 days ago...

Reply:
Sorry, missed that. [As you asked](https://github.com/Galaxia5987/robot-2021/pull/8#commitcomment-45818341), it's a bit hard to follow comments written on the PR. Please tag if you do, so I'll receive a notification.